unit-tests/catch.h

```diff
@@ -9,8 +9,13 @@
 // Otherwise, don't let it define its own:
 #define ELPP_NO_CHECK_MACROS
 
-#include "catch/catch.hpp"
+#if defined(CATCH_CONFIG_MAIN)
+#error CATCH_CONFIG_MAIN defined
+#endif
 
+#include "catch/catch.hpp"
+#include <string>
+std::string context = "";
```

Comment:
Remove ` = ""`
I'd also maybe try to use a name that's more obvious... `context` can be confused with `librealsense::context` and maybe just a local variable. Maybe `test::context` to agree with `test.context` in Python?